packages/core/src/types.ts

```diff
@@ -13,10 +13,10 @@ export class ActionResult {
     static fromContext(ctx: Context) {
         return new ActionResult(ctx.body, ctx.status)
     }
-    private readonly headers: { [key: string]: string } = {}
+    private readonly headers: { [key: string]: string[] } = {}
```

Comment:
Please change the type from `string[]` to `string | string[]` so it will not break previous version

Reply:
lol, I don't know you could do that, I'm on it.